src/driver/mod.rs

```diff
@@ -39,6 +39,8 @@ mod write;
 
 mod writev;
 
+use crate::driver::op::Lifecycle;
+use io_uring::opcode::AsyncCancel;
```

Comment:
The async cancel op has been around since Oct 2019. I don't know what kernel version that was, but we should probably find some base version we want to support and verify the uring interface version through a series of tests when the driver is first created. Else we could be running this on an old kernel where the async_cancel isn't supported and we wouldn't see the errors returned because we're ignoring the results of the cancel operation.

So maybe nothing to really fix here. Maybe a comment saying we want to test for this operation in the future.

Reply:
There isn't really a way to get sane shutdown without this, and IMO Bullseye should be the oldest thing we support for the core APIs.